packages/svelte/package.json

```
@@ -9,6 +9,7 @@
     "node": ">=18"
   },
   "files": [
+    "!src/**/*.test.*",
     "src",
```

I think the negation needs to happen afterwards (maybe it doesn't matter, but it also reads better that way)
```suggestion
    "src",
    "!src/**/*.test.*",
```